src/App.tsx

```diff
@@ -1,12 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, getRed, get5First } from './api/goods';
+import { GoodsList } from './components/GoodsList';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+
+  const all = () => {
+    getAll().then(allGoods => setGoods(allGoods));
```

Comment:
it'll be great to provide for error

Reply:
added catch with alert. Let me know, if You want anything more fancy.

Reply:
Nothing, by now)